core/unix/os_exports.h

```diff
@@ -200,7 +200,7 @@ extern uint android_tls_base_offs;
  *     void *private;
  *   } tcb_head_t;
  */
```

Comment:
The comments should also be updated.

Reply:
I thought about it, but it still, sort of, skips dtv by jumping only 4/8 bytes, instead of the beginning of the TCB (8/16). Perhaps something like /* point to tcb->private, skipping dtv */ would be more appropriate?

Reply:
Yeah I think it’s fine. But what I mean is the comment above:

```
/* FIXME i#3544: We might need to re-use ARM's approach and store DR TLS in
 * tcb_head_t::private field: typedef struct
 *   {
 *     dtv_t *dtv;
  *     void *private;
  *   } tcb_head_t;
  */
```

Reply:
How about:
/* Re-using ARM's approach and store DR TLS in tcb_head_t::private, 
* with the only difference being tp register points at the end of TCB
* typedef struct
*   {
 *     dtv_t *dtv;
  *     void *private;
  *   } tcb_head_t;
  */

Reply:
I'm fine with it.